src/pageobjects/workbench/Input.ts

```diff
@@ -45,7 +45,7 @@ export abstract class Input extends BasePage<AllInputLocators> {
 
         // fallback to clipboard if the text gets malformed
         const currentText = await this.getText()
-        if (currentText !== text) {
+        if (currentText && currentText !== text) {
```

Comment:
My assumption here is that if there is no text we don't have to enter any backspaces or mess with the clipboard but happy to be corrected.